jwql/website/apps/jwql/archive_database_update.py

```diff
@@ -20,11 +20,15 @@
         instrument = 'nircam'
         get_updates(insturument)
 
-        this module can be run from command line to add new elements to the django database
+        can be run from command line to add new elements to the django database
         $ python archive_database_update.py
 
-        This module can use the 'update' argument to update every rootfileinfo data model with the most complete information from MAST
-        $ python archive_database_update.py update
+        Use the '--fill_empty' argument to provide a model and field.  Updates ALL fields for any model with empty/null/0 specified field
+        $ python archive_database_update.py --fill_empty root_file_info Filter
+
```

Comment:
Thanks!  I ran this example locally, and it found 9082 changes to make, but the database didn't seem to be changing.  It looks like there are large numbers of files that actually don't have filter values -- all the MIRI IFU files and at least some FGS files have no FILTER keyword in the FITS headers.

I just canceled that run and ran with expstart instead of filter.  That appropriately found 23 FGS files that needed updating, and fixed them.

So, maybe one more change to the example, to try expstart instead of filter?  But the code looks like it's working!

Reply:
Thats great feedback.  I remembered to put the specific fields to use in this PR, but I should also probably include that tidbit in the actual docstring.

Reply:
And done